charts/victoria-metrics-agent/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Next release
 
+**Update note**: `remoteWriteUrls` and `multiTenantUrls` were deprecated and replaced by `remoteWrite`.
```

Comment:
Based on code changes an update note should be:

```suggestion
**Update note**: `remoteWriteUrls` and `multiTenantUrls` parameters were replaced by `remoteWrite`.
```

Deprecation is not appliable here since parameters are now removed in favor of a new one. 

Wdyt about adding a section to a readme called "upgrade guide"?
This change can be a first item to note there. We can put a short guidance like update X to Y so that users won't have to check an updated version of `values.yaml`.
Alternatively, we can do the same in changelog but it will be easier find multiple pending changes when upgrading between multiple versions if such guides would be at one place.

Reply:
@zekker6 updated PR, changed upgrade guide